_rules/meta-viewport-b4f0c3.md

```diff
@@ -68,7 +68,7 @@ The `meta name="viewport"` element does not define the `maximum-scale` and `user
 
 #### Passed Example 2
 
-The `meta name="viewport"` element defines the `user-scalable=yes` so the user can still zoom in.
+This `meta` element with a `name="viewport"` and `content` attributes specifies `user-scalable=yes` and does not specify the `maximum-scale` value.
```

Comment:
I think you've inverted cause and effect on these. Try expressing them like this instead:

```suggestion
This  viewport `meta` element does not prevent user scaling because it has `user-scalable` set to `yes`.
```

The same is true for the rest of the example descriptions. Please see what you can do to improve them.